packages/open-api-page/fixtures/refresh-generated-fixtures.js

```diff
@@ -0,0 +1,39 @@
+import fs from 'fs'
+import packerRawSwagger from './raw-schemas/packer.swagger.json'
+import boundaryRawSwagger from './raw-schemas/boundary.swagger.json'
+import processSchema from '../process-schema'
+import { getPropsForPage } from '../server'
+
+main()
```

Comment:
This is a dev-targeted utility function for generating processed fixtures that are easy to import into Swingset.